### PR TITLE
Clear console branch

### LINKED ISSUE #120: [UI/UX] Clear console button
The console becomes very long, we should add a way to clear all the text

### PR DESCRIPTION
**Pull request by Thine-Sho**
#120

### CODE REVIEW

**Comment by nia-n** [comment]
Great job with functionality! Couple notes on style: capitalize text on button, change wording to keep consistent with name of tab ("clear log" -> "Clear Console"). More minor note: try getting the button to look more similar to the "Upload Code" and "Upload Field" buttons (Have it "float" in the grayish bar instead of going all the way to the bottom of the bar like a tab. Align the button on the right side of the bar, similar to the position of the upload buttons)

File: static/js/base.js [hunk 1]
```diff
@@ -549,3 +549,8 @@ function log(text) {
     }
     consoleLog.scrollTop = consoleLog.scrollHeight;
 }
+
+let clear_console = () => {
```
Comment (chaoryan5) [inline static/js/base.js]: could make a traditional function i.e. `function clear_console()`

also change to snake case i.e. `clearConsole`

File: templates/index.html [hunk 2]
```diff
@@ -180,9 +180,9 @@ <h5 class="modal-title">April 26, 2021 Updates</h5>
                 <!-- Simulator Info -->
                 <div class="bd-highlight flex-fill">
                     <div class="card border-primary h-100">
-                        <div class="card-header" style="padding-top:7px">
+                        <div class="card-header d-flex align-items-end" style="margin-bottom:0px; padding:0px!important;">
```
Comment (chaoryan5) [inline templates/index.html]: to keep things consistent, remove spaces in style string (also for line 185)

File: templates/index.html [hunk 3]
```diff
@@ -192,12 +192,35 @@ <h5 class="modal-title">April 26, 2021 Updates</h5>
...
+
+                              <div class="p-2">
+                                <button class="nav-item-clear-console btn btn-success" id="console-clear" onclick="clear_console();">
+                                    <span>Clear Console</span>
+                                </button>
+                                <script>
+                                  let nav_item = document.querySelectorAll('#set-tabs .nav-item');
+                                  let clear_btn = document.getElementById("console-clear");
+
+                                  nav_item.forEach((li)=>{
```
Comment (chaoryan5) [inline templates/index.html]: can space things out to look a little more clean, like
`(li) => {`
`} else {`

**Review by chaoryan5 — COMMENTED** [review]
good job, just a couple stylistic things to fix

**Review by nia-n — APPROVED** [review]
Great job!

**Pull request merged**